mixer/pkg/api/grpcServer.go

```diff
@@ -338,11 +292,11 @@ func (s *grpcServer) Report(legacyCtx legacyContext.Context, req *mixerpb.Report
 
 		span.LogFields(otlog.String("success", "finished Report for attribute bag "+string(i)))
 		span.Finish()
-		preprocResponseBag.Reset()
+		reportBag.Reset()
```

Comment:
A comment about why the reset of the reportBag is needed would be very helpful. It is hard to see that the bag is getting re-seeded with APA attributes.